public/app/percona/backup/components/BackupInventory/BackupInventory.tsx

```diff
@@ -63,16 +66,16 @@ export const BackupInventory: FC = () => {
   );
   const styles = useStyles(getStyles);
 
-  const getData = async () => {
-    setPending(true);
+  const getData = async (showLoading = false) => {
+    showLoading && setPending(true);
```

Comment:
This is not a state variable, it's just a flag we pass as true on the first request, so that the spinner won't show up on subsequent requests (related to what's mentioned above - similar to what's in `DBCluster.hooks.ts`, please check it.

Reply:
Yes, my point is that we could inhibit the loader from showing in the tsx code instead of doing it here. It might make the code more readable, since it's not immediate how you're using it here, but it'd clutter the tsx code.

It's ok to keep it here, but you don't need the check for `setPending(false)`

Reply:
👍